zh-CN/mapping_dev.zh_CN

```diff
@@ -590,173 +590,173 @@ dlgSgCustomizeProjectUi.mni"Regular Expressions"=常用表达式
+dlgSgFileCompareNoChanges.chk=不再显示
+dlgSgFileCompareNoChanges.fur=两个文件内容按字节顺序相等。\n要查看文件内容，请单击 “打开” 。
+dlgSgFileCompareNoChanges.hdl=打开文件比较，但不显示任何更改？
+dlgSgFileCompareNoChanges.tle=文件比较
 dlgSgFindObject.edt"Repository Path, Commit ID or Ref"=\
- Repository Path, Commit ID or Ref
+ 仓库路径、提交 ID 或参考
 dlgSgFindObject.tle=查找对象
-dlgSgFlowBranchDivergedHandlerFastForward.btn"Fast-Forward"=Fast-Forward
-dlgSgFlowBranchDivergedHandlerFastForward.fur%3=The local branch '$1' is behind its tracked branch '$2'. You may fast-forward now or do it manually later, e.g. by checking out the branch '$3'.
-dlgSgFlowBranchDivergedHandlerFastForward.hdl%2=Should branch '$1' be fast-forwarded to '$2'?
-dlgSgFlowBranchDivergedHandlerFastForward.tle=Start Feature
-dlgSgFlowBranchDivergedHandlerReplaceRemote.btn"Replace"=Replace
-dlgSgFlowBranchDivergedHandlerReplaceRemote.fur%2=The local branch '$1' seems to contain more recent but rewritten commits of remote branch '$2'.\n\nIf you are not sure whether the local branch is actually more recent than the remote branch, you should better cancel this operation and investigate local and remote changes in more detail.
-dlgSgFlowBranchDivergedHandlerReplaceRemote.hdl%2=Should branch '$1' replace remote branch '$2'?
-dlgSgFlowBranchDivergedHandlerReplaceRemote.tle=Finish Feature
+dlgSgFlowBranchDivergedHandlerFastForward.btn"Fast-Forward"=快进
+dlgSgFlowBranchDivergedHandlerFastForward.fur%3=本地分支 “$1” 位于其跟踪分支 “$2” 的后面。您可以现在快进，也可以稍后手动前进，例如签出分支 $3 。
+dlgSgFlowBranchDivergedHandlerFastForward.hdl%2=分支 “$1” 是否应快速转发到 “$2”？
+dlgSgFlowBranchDivergedHandlerFastForward.tle=开始 feature
+dlgSgFlowBranchDivergedHandlerReplaceRemote.btn"Replace"=替换
+dlgSgFlowBranchDivergedHandlerReplaceRemote.fur%2=本地分支 “$1” 似乎包含远程分支 “$2” 的更新但已重写的提交。\n\n如果您不确定本地分支是否实际比远程分支更新，则最好取消此操作并更详细地调查本地和远程更改。
+dlgSgFlowBranchDivergedHandlerReplaceRemote.hdl%2=分支 “$1” 是否应替换远程分支 “$2”？
+dlgSgFlowBranchDivergedHandlerReplaceRemote.tle=完成 feature
 dlgSgFlowConfigure.btn"Reset to Defaults"=重置为默认
-dlgSgFlowConfigure.edt"Develop Branch"=Develop分支
-dlgSgFlowConfigure.edt"Feature Branches"=Feature分支
-dlgSgFlowConfigure.edt"Git-Flow Type"=Git工作流类型
-dlgSgFlowConfigure.edt"Hot-Fix Branches"="Hot-Fix分支
-dlgSgFlowConfigure.edt"Main Development Branch"="Main Develop分支
-dlgSgFlowConfigure.edt"Master Branch"=Master分支
+dlgSgFlowConfigure.edt"Develop Branch"=Develop 分支
+dlgSgFlowConfigure.edt"Feature Branches"=Feature 分支
+dlgSgFlowConfigure.edt"Git-Flow Type"=Git 工作流类型
+dlgSgFlowConfigure.edt"Hot-Fix Branches"="Hot-Fix 分支
+dlgSgFlowConfigure.edt"Main Development Branch"="主开发分支
+dlgSgFlowConfigure.edt"Master Branch"=Master 分支
 dlgSgFlowConfigure.edt"Prefix for Feature Branches"=功能分支的前缀
-dlgSgFlowConfigure.edt"Release Branches"=Release Branches
-dlgSgFlowConfigure.edt"Remote"=Remote
-dlgSgFlowConfigure.edt"Support Branches"=Support Branches
+dlgSgFlowConfigure.edt"Release Branches"=发布分支
+dlgSgFlowConfigure.edt"Remote"=远程
+dlgSgFlowConfigure.edt"Support Branches"=支持分支
```

Comment:
"Release Branches" and  "Support Branches", the keywords "Release" "Support" should be keeped.  Need to be consistent with the above ("Develop Branch", "Master Branch").   Do you think?

Reply:
I agree.Thanks, I've fixed it.